app/gui/src/presenter/searcher.rs

```diff
@@ -190,24 +191,43 @@ impl Model {
         }
     }
 
+    fn breadcrumb_selected(&self, id: BreadcrumbId) {
+        self.controller.select_breadcrumb(id);
+    }
+
+    fn set_breadcrumbs(&self, names: impl Iterator<Item = ImString>) {
+        if let SearcherVariant::ComponentBrowser(browser) = self.view.searcher() {
+            browser.model().list.set_breadcrumbs_from((names.map(Into::into).collect(), 1));
```

Comment:
What is this `1`?

Reply:
The index to update the breadcrumbs from. Refactored and added a comment